app/helpers/platform_admin_vendors_helper.rb

```diff
@@ -0,0 +1,9 @@
+module PlatformAdminVendorsHelper
+  def pending_vendors(vendors)
+    vendors.where(status: "pending")
+  end
```

Comment:
Can't we keep this in the Vendors model? Something like:

def self.pending
  where(status: "pending")
end

def self.inactive
  where(status: "inactive")
end

Then we could call it from the views using @vendors.pending and @vendors.inactive. We could have a similar method for @vendors.active